mkdocs.yml

```diff
@@ -10,6 +10,7 @@ nav:
     - Overview: README.md
     - Setup:
       - Build from source: building.md
+      - Build with Docker: docker.md
```

Comment:
@brotzeit moved the relevant documentation here, is that ok?

Reply:
Yeah that's fine.